install/kubernetes/cilium/values.yaml

```diff
@@ -295,6 +295,9 @@ global:
   # kubeProxyReplacement enables kube-proxy replacement in Cilium BPF datapath
   kubeProxyReplacement: "probe"
 
+  # kubeProxyReplacement healthz server bind address
+  kubeProxyReplacementHealthzBindAddr: "0.0.0.0:10256"
```

Comment:
Can you add this to `install/kubernetes/cilium/charts/config/values.yaml` instead?

Reply:
Sure will move this.